src/java.base/share/classes/java/security/PrivilegedActionException.java

```diff
@@ -98,21 +98,25 @@ public String toString() {
     /**
      * Serializable fields for UndeclaredThrowableException.
```

Comment:
The "UndeclaredThrowableException" is confusing and not defined anywhere in this class specification. Suggest changing this to: 

Serializable fields for this {@code PrivilegedActionException}.